### PR TITLE
Config + specs and an update to the readme

### PR DESCRIPTION
**Pull request by film42**
@mmmries @quixoten @abrandoned

### CODE REVIEW

**Comment by film42** [comment]
Weird.. they're passing locally.

**Comment by film42** [comment]
Working and ready for 👀 !

File: spec/protobuf/nats/config_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    subject.load_from_yml
+    expect(subject.servers).to eq(["nats://127.0.0.1:4222", "nats://127.0.0.1:4223", "nats://127.0.0.1:4224"])
+    expect(subject.uses_tls).to eq(true)
+    expect(subject.connect_timeout).to eq(2)
+
+    ENV["PROTOBUF_NATS_CONFIG_PATH"] = nil
+  end
+
+  it "loads the defaulss when a yml config is missing" do
```
Comment (quixoten) [inline spec/protobuf/nats/config_spec.rb]: defaulss!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (film42) [inline spec/protobuf/nats/config_spec.rb]: fixed cause i gotta pad them stats boiiiiiiiiiii

**Pull request merged**